pkg/bootstrap/docker/run/run.go

```diff
@@ -112,7 +125,11 @@ func (h *Runner) Env(env ...string) *Runner {
 
 // Privileged tells Docker to run the container as privileged
 func (h *Runner) Privileged() *Runner {
```

Comment:
I remember that Docker doesn't allow `--privileged` option so I thought that API also doesn't accept it. Ok, I'll try it on Monday.